lightning/src/ln/msgs.rs

```diff
@@ -394,7 +394,8 @@ pub struct UnsignedNodeAnnouncement {
 /// A node_announcement message to be sent or received from a peer
 pub struct NodeAnnouncement {
 	pub(crate) signature: Signature,
-	pub(crate) contents: UnsignedNodeAnnouncement,
+	/// The actual node announcement
```

Comment:
I think we should totally do this, though it'd be nice to just do it for pretty much every message at once.

Reply:
Are you saying that all fields of all messages should be pub?

Reply:
I faintly remember that we did discuss making them all pub.  What level of inline documentation do we want?  just what each field is, or also a quick statement of how it's used?

Reply:
I think just what the field is for now. How it's consumed would be really a lot of docs for some if you to reach clarity. Documentation stuffing can be done as a follow-up.

Reply:
Should I move this to a new PR that includes everything?

Reply:
Moved to #665